app/helpers/record_helper.rb

```diff
@@ -128,4 +128,19 @@ def restricted_link?
   def guest_and_restricted_link?
     guest? && restricted_link?
   end
+
+  # force http images to use https. If remote server cannot handle https a
+  # broken image is expected
```

Comment:
Is this also expected *by UX* and they're cool with that?

Reply:
lol. I thought so but I will confirm.